local/test-binary

```diff
@@ -93,45 +103,42 @@ if [[ "${DEFAULT_TESTING}" != "no" ]]; then
     process_mtr_output MySQL "${WORKDIR_ABS}/mtr.output" "${WORKDIR_ABS}/junit.xml"
 fi
 
-
+# we test with system jemalloc
```

Comment:
Why?

Reply:
because we need jemalloc only for tokudb tests and not for regular tests

Reply:
No harm (and potential benefit) to enable jemalloc for regular tests too

Reply:
added